app/helpers/pg_hero/home_helper.rb

```diff
@@ -15,5 +15,15 @@ def pghero_pretty_ident(table, schema: nil)
     def pghero_js_var(name, value)
       "var #{name} = #{json_escape(value.to_json(root: false))};".html_safe
     end
+
+    def pghero_remove_index(query)
+      if query[:columns]
+        columns = query[:columns].map(&:to_sym)
+        columns = columns.first if columns.size == 1
+      end
+      ret = "remove_index #{ query[:table].to_sym.inspect },"
+      ret << "\n              column: #{ columns.inspect},\n             " if columns
+      ret << " name: #{ (query[:name] || query[:index]).to_s.inspect }"
```

Comment:
Minor suggestion: Can we keep each `remove_index` call on a single line and move the `column` option to the end?